pkg/v3/tickers/recovery.go

```diff
@@ -8,6 +8,18 @@ import (
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
 )
 
+const (
+	DefaultRecoveryDelay           = 1 * time.Minute
+	DefaultMaxRecoveryDuration     = 60 * time.Minute
```

Comment:
should we make these configurable through env vars? I'm nervous we'll want to try to change these at some point and if they're hard-coded it will have to be via a release.